blocks/editable/index.js

```diff
@@ -723,3 +743,11 @@ Editable.defaultProps = {
 	formattingControls: DEFAULT_FORMATS,
 	formatters: [],
 };
+
+Editable.Value = ( { tagName: TagName = 'div', value = [], ...props } ) => {
+	const HTML = createHTMLFromSimpleNodeList( value );
+
+	return (
+		<TagName dangerouslySetInnerHTML={ { __html: HTML } } { ...props } />
```

Comment:
We should omit `tagName` and `value` here.

We can also move `const TagName = tagName.toLowerCase()` in here to simplify usage.